src/Archive.AspNetCore.All/Archive.AspNetCore.All.csproj

```diff
@@ -1,5 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
+  <Import Project="..\..\build\dependencies.props" />
```

Comment:
It seems like we need this after https://github.com/aspnet/MetaPackages/commit/0b844b2ccf76fcd01c25dfaf7d4b9c417a84b38a.